src/DataBlobs/services/BlobStores.jl

```diff
@@ -94,21 +98,46 @@ function getBlob(dfg::AbstractDFG, entry::BlobEntry)
     end
     throw(
         KeyError(
-            "could not find $(entry.label), uuid $(entry.id)) in any of the listed blobstores:\n $([s->getKey(s) for (s,v) in stores]))"
+            "could not find $(entry.label), uuid $blobId) in any of the listed blobstores:\n $([s->getKey(s) for (s,v) in stores]))"
         )
     )
 end
 
-addBlob!(dfg::AbstractDFG, entry::BlobEntry, data::T) where T =
-        addBlob!(getBlobStore(dfg, entry.blobstore), entry, data)
+getBlob(store::AbstractBlobStore, entry::BlobEntry) =
+    getBlob(store, entry.originId)
```

Comment:
not sure if you can assume originId here.  May have to look for both blobId first or fall back on originId second.

Reply:
Ok, My first aim was just to get all the originId vs blobId calls to go through one place to fix the places that used entry.id. 
I'm not sure of the use of origin vs blob so will change as you say only use originId if blobId is nothing